### PR TITLE
uploading pics(help needed)

### PR DESCRIPTION
**Pull request by 0xAlison**

### CODE REVIEW

**Comment by 0xAlison** [comment]
1. I have added two properties into class User. The first one is "pic" which indicates the pic url(type: string). The second one is "actionUrl" which indicates the action property of form html label(this is not important and it has been completed as well)
2. I've added the pic uploading box on the user-info.html. so as the submit button pushed, it will be uploaded. I use blobstore to generate the url of the pic we had uploaded.
3. so the problem now is I can't store the "pic"(pic url) property into datastore. I've tested on every interface and i think the problem is that the servlet couldn't spot the image uploading box(like the pic below, the servlet couldn't spot userData. i think it's due to some property of vuetify. call vuetify master!) the url generating function works pretty well (already tested!  
image

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -117,7 +117,13 @@ function fetchAboutMe() {
         indexVue.user_teachcate=info.teachCate;
         indexVue.user_level=info.skillLevel;
         indexVue.user_school=info.school;
-        document.getElementById('about-me-container').innerHTML=info.aboutMe;
+        document.getElementById('picc').src = info.pic;
```
Comment (nicolercl) [inline src/main/webapp/js/user-page-loader.js:120]: Hi Alison, there is no html element with id = "picc" or "url" in the user-page.html. This seems to be the problem.

**Comment by Sinyu104** [comment]
Has the problem been solved?

**Comment by 0xAlison** [comment]
not yet:( still trying 0.0

**Pull request closed without merging**